src/main/java/UseCaseClasses/UserList.java

```diff
@@ -6,25 +6,25 @@
 
 public class UserList {
 
-    public HashMap<String, Person> users;
```

Comment:
see earlier comment about static methods

Reply:
this should be fixable again with a specific instance of UserList -- to initialize a new UserList and then call on an instance of InformationSaver (use retrieveUsers or retrieveIncomingRequests or something to get all the users / requests we currently have in the database)